lib/rubocop/extension/generator/generator.rb

```diff
@@ -137,7 +137,11 @@ def self.defaults!
             gem 'rspec'
           RUBY
 
-          patch 'README.md', /^gem '#{name}'$/, "gem '#{name}', require: false"
+          if Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.3.9')
+            patch 'README.md', /\$ bundle add #{name}$/, "\$ bundle add #{name} --require=false"
+          else
+            patch 'README.md', /^gem '#{name}'$/, "gem '#{name}', require: false"
+          end
```

Comment:
Multi-line does not seem to be expected, right?

```suggestion
          if Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.3.9')
            patch 'README.md', /\$ bundle add #{name}\z/, "\$ bundle add #{name} --require=false"
          else
            patch 'README.md', /\Agem '#{name}'\z/, "gem '#{name}', require: false"
          end
```

Reply:
Exactly.

Reply:
Sorry I've committed this without thinking much but the regex incorrect because `\$ bundle add #{name}` is a line, part of `README.md`. I'll revert the commit.